app/presenters/finder_presenter.rb

```diff
@@ -4,6 +4,7 @@ class FinderPresenter
 
   delegate :beta_message,
            :document_noun,
+           :context,
```

Comment:
Its use in the component is implicitly scoped to being about the title because it's the `govuk_component/title` partial.

What sort of content are you intending this to be used for? It feels like there should either be a more descriptive name to use for this, or we should be using something like `title_context` and update `name` to be `title`.